### ISSUE TITLE
"Color highlight extension" dosn't work will with "vscode-live-sass-compiler extension"

### ISSUE THREAD

**Issue by moghaazi** [egonyans/vscode-ext-color-highlight#93]
Every time I write **@import** in **sass** file, I'm receiving the message "**Extension host terminated unexpectedly**".
To fix this issue, I wrote this in settings.json `"color-highlight.languages": [ "*", "!scss" ]`,
But that makes **"Color highlight extension"** doesn't work in sass file

**Comment by mario-cvetkovski** [comment]
Very useful extension, I really like it. But I cannot use it because I got this error "Extension host terminated unexpectedly" when I am using "Live Sass Compiler" extension. I hope you can fix this issue in the future.

**Comment by GBrachetta** [comment]
Nobody seems to read the issues on this repo. I have the same issue (and a few other hundred thousands).

**Comment by moghaazi** [comment]
> Nobody seems to read the issues on this repo. I have the same issue (and a few other hundred thousands).

Yes many repositories like that, they don't care about the issues

**Comment by GBrachetta** [comment]
> > Nobody seems to read the issues on this repo. I have the same issue (and a few other hundred thousands).
> 
> Yes many repositories like that, they don't care about the issues

Pity, because otherwise the extension is pretty cool and specially useful with sass and variables.
